lib/adapters/MochaAdapter.js

```diff
@@ -52,10 +52,25 @@ export default class MochaAdapter extends EventEmitter {
     // a "test start".
     if (mochaTest.errors !== undefined) {
       var status = (mochaTest.state === undefined) ? 'skipped' : mochaTest.state
-
-      // Test end.
+      let errors = []
+
+      // Mocha has indeed only one error per test, but by this is avoided the
```

Comment:
I wanted to make clear why I used `forEach`, although that Mocha has only one error. I will delete I think.